editor/import/resource_importer_texture.cpp

```diff
@@ -380,7 +380,7 @@ void ResourceImporterTexture::_save_ctex(const Ref<Image> &p_image, const String
 	f->store_32(0);
 	f->store_32(0);
 
-	if ((p_compress_mode == COMPRESS_LOSSLESS || p_compress_mode == COMPRESS_LOSSY) && p_image->get_format() > Image::FORMAT_RGBA8) {
+	if ((p_compress_mode == COMPRESS_LOSSLESS || p_compress_mode == COMPRESS_LOSSY) && p_image->get_format() >= Image::FORMAT_RF) {
```

Comment:
Note for the future, this changes the behaviour for `FORMAT_RGBA4444` and `FORMAT_RGB565`. 

I think the change makes sense as people who intentionally use such compressed formats would be surprised that Godot converts them to an RGBA8 internally. Making note of it here though as it is a change in behaviour.